internal/pkg/enccid/enc_bigint.go

```diff
@@ -0,0 +1,31 @@
+package enccid
+
+import (
+	"math/big"
+)
+
+// EncBigInt is an encodable big integer
+type EncBigInt struct {
+	i *big.Int
+}
+
+// NewEncBigInt creates a new encodable big int from a golang stdlib big int
+func NewEncBigInt(i *big.Int) *EncBigInt {
+	return &EncBigInt{i}
+}
+
+// ToBigInt converts an encodable big int to a golang stdlib big int
+func (b EncBigInt) ToBigInt() *big.Int {
+	return b.i
+}
+
+// MarshalBinary returns the bytes of a big int
+func (b EncBigInt) MarshalBinary() ([]byte, error) {
+	return b.i.Bytes(), nil
+}
+
+// UnmarshalBinary parses a big int from bytes
+func (b *EncBigInt) UnmarshalBinary(x []byte) error {
+	b.i = big.NewInt(0).SetBytes(x)
+	return nil
+}
```

Comment:
The specs-actors repo includes a big int type, and we should replace all our encodable uses with that.

Reply:
👍 Will defer to the next PR I do addressing all of our big int technical debt.